src/app/helptext/jails/jails-add.ts

```diff
@@ -362,7 +363,9 @@ IPv4, IPv6, local (UNIX), and route. <br>\
 for all protocal stacks.'),
 
 vnet_interfaces_placeholder: T('vnet.interfaces'),
-vnet_interfaces_tooltip: T(''),
+vnet_interfaces_tooltip: T('A space delimited list of network \
+interfaces for a VNET-enabled jail after it is created. \
```

Comment:
Not sure what "after it is created" means here.  Or "for".  Is this a list of VNET interfaces that are allowed or attached to the jail?